cmd/convert_nss_certdata_to_pems/README.md

```diff
@@ -0,0 +1,6 @@
+
+Usage:
+```
+go get github.com/jcjones/nss-tools/cmd/convert_nss_certdata_to_pems
```

Comment:
Do you want to change this to this new repo?

Reply:
mmm, I do. Blagh. Missed the ping.